converter/index.html

```diff
@@ -1,39 +1,17 @@
 <!DOCTYPE HTML>
 <head>
     <title>temperature converter</title>
-    <h1 style="text-align: center">Temperature Converter</h1>
     <link rel="stylesheet" type="text/css" href="style.css"/>
-    <!-- <style>
-        #box {
-            width: 100px;
-            height: 30px;
-            border-style: solid;
-            text-align: center;
-        }
-    </style> -->
+    <script src="script.js"></script>
```

Comment:
This is fine in this case but usually you'll want to put JS files at the end of your body, right before the closing `</body>` tag. This is because some times your JS code will be waiting for the DOM elements to be available and your document to be ready to act on it. We'll explore more of that in other places.